roofit/hs3/inc/RooFitHS3/RooJSONFactoryWSTool.h

```diff
@@ -32,16 +33,27 @@ class RooJSONFactoryWSTool {
    };
    class Exporter {
    public:
+      virtual std::string key() const = 0;
       virtual bool autoExportDependants() const { return true; }
       virtual bool exportObject(RooJSONFactoryWSTool *, const RooAbsArg *, RooFit::Detail::JSONNode &) const
       {
          return false;
       }
       virtual ~Exporter(){};
    };
+   struct ExportKeys {
+      std::string type;
+      std::map<std::string, std::string> proxies;
+   };
+   struct ImportExpression {
+      TClass *tclass;
```

Comment:
```suggestion
      TClass *tclass = nullptr;
```
Just a good practice for raw pointer members, so we don't risk getting invalid pointers.